docs/rules/prefix-selectors-with-select.md

```diff
@@ -30,6 +40,8 @@ export const selectFeature = createSelector((state: AppState) => state.feature)
 export const selectFeature: MemoizedSelector<any, any> = (state: AppState) =>
   state.feature
 const selectFeature = createFeatureSelector<FeatureState>(featureKey)
+export const selectThing = (id: string) =>
```

Comment:
Nit, to keep things inline we should add linebreaks between the correct examples 😅


```suggestion

export const selectThing = (id: string) =>
```